Badge documentation (#1956)
telerik/blazor-docs: components/badge/appearance.md

---
title: Appearance
page_title: Badge Appearance

> description: Appearance settings of the Badge for Blazor. See the available options that allow you to fully customize the look of the Badge component. 

Reviewer: ```suggestion
description: Explore the appearance settings of the Badge for Blazor. See the available options that allow you to fully customize the look of the Badge component. 
```

slug: badge-appearance
tags: telerik,blazor,badge,navbar,appearance
published: True
position: 35
---

# Appearance Settings

> This article outlines the available Badge parameters, which control its appearance.

Reviewer: ```suggestion
The Badge component features built-in appearance parameters that allow you to customize virtually every aspect of its look and feel.
```

> The Badge component features built-in appearance parameters that allow you to customize virtually every aspect of its look and feel.

Reviewer 2: "Every aspect" is bit of an overstatement :)

```suggestion
The Badge component features built-in appearance parameters that allow you to customize multiple aspects of its look and feel.
```

## FillMode

> You can control how the Badge is filled by setting the `FillMode` parameter to a member of the `Telerik.Blazor.ThemeConstants.Badge.FillMode` class:

Reviewer 2: Avoid explaining a term with the same keyword.

```suggestion
You can toggle the Badge border and background by setting the `FillMode` parameter to a member of the `Telerik.Blazor.ThemeConstants.Badge.FillMode` class:
```

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Class members | Manual declarations |
|---------------|--------|
| `Solid` (default value) | `solid`   |
| `Flat` | `flat`|
| `Outline` | `outline`|

> See the [example](#example) below to customize the available parameters and see how they affect the Badge component.

Reviewer: ```suggestion
Refer to the [example](#example) below to customize the available parameters and observe their impact on the Badge component.
```

## Rounded

> The `Rounded` parameter applies the `border-radius` CSS rule to the Badge to achieve curving of the edges.  You can set it to a member of the `Telerik.Blazor.ThemeConstants.Badge.Rounded` class:

Reviewer: ```suggestion
The `Rounded` parameter applies the `border-radius` CSS rule to the Badge and lets you curve its edges. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Badge.Rounded` class:
```

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Class members | Manual declarations |
|---------------|--------|
| `Small` | `sm`   |
| `Medium` | `md`|
| `Large` | `lg`|
| `Full` (default value) | `full`|

> See the [example](#example) below to customize the available parameters and see how they affect the Badge component.

Reviewer: ```suggestion
Refer to the [example](#example) below to customize the available parameters and observe their impact on the Badge component.
```

## Size

You can increase or decrease the size of the Badge by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Badge.Size` class:

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Class members | Manual declarations |
|---------------|--------|
| `Small` | `sm`   |
| `Medium` (default value) | `md`|
| `Large` | `lg`|

> See the [example](#example) below to customize the available parameters and see how they affect the Badge component.

Reviewer: ```suggestion
Refer to the [example](#example) below to customize the available parameters and observe their impact on the Badge component.
```

## ThemeColor

You can change the color of the Badge by setting the `ThemeColor` parameter to a member of the `Telerik.Blazor.ThemeConstants.Badge.ThemeColor` class:

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Class members | Manual declarations |
|---------------|--------|
| `Base` | `base`   |
| `Primary` | `primary`|
| `Secondary` | `secondary`|
| `Tertiary` | `tertiary`|
| `Info` | `info`   |
| `Success` | `success`|
| `Warning` | `warning`|
| `Error` | `error`  |
| `Dark` | `dark`   |
| `Light` | `light`  |
| `Inverse` | `inverse`|

> See the [example](#example) below to customize the available parameters and see how they affect the Badge component.

Reviewer: ```suggestion
Refer to the [example](#example) below to customize the available parameters and observe their impact on the Badge component.
```
